app/configs/locationCrashesTable.ts

```diff
@@ -0,0 +1,81 @@
+import { locationCrashesColumns } from "./locationCrashesColumns";
+import { QueryConfig, FilterGroup } from "@/utils/queryBuilder";
+import { DEFAULT_QUERY_LIMIT } from "@/utils/constants";
+import {
+  getStartOfYearDate,
+  makeDateFilters,
+} from "@/components/TableDateSelector";
+
+const columns = locationCrashesColumns.map((col) => String(col.path));
+
+const locationCrashesFiltercards: FilterGroup[] = [
```

Comment:
the old VZE has a lot of advanced filters on the cr3 crashes table. i'm a bit skeptical we need all of them. i will check with Xavier.


i'd also like to check in with some of our non-VZ users who access this page. we have a few power users identified from the ACC research.